specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2018-02-01-preview/examples/BuildDefinitionsDelete.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "subscriptionId": "4385cf00-2d3a-425a-832f-f4285b1c9dce",
+    "resourceGroupName": "myResourceGroup",
+    "registryName": "myRegistry",
+    "api-version": "2018-02-01-preview",
+    "buildDefinitionName": "myBuildDefinition"
+  },
+  "responses": {
+    "200": {},
+    "202": {},
+    "404": {}
```

Comment:
`204`?

Reply:
Ideally delete should not throw 404. if there is no content, it should be 204.